### PR TITLE
DA/reactions buttons

### PR DESCRIPTION
**Pull request by d-atkins**
## Description
Completes [User Story 17: Post Show](https://app.asana.com/0/1170154762648575/1170778683506483/f)

## Notes
Small PR that adds buttons to a post's show page.

Refactors/reorganizes a few tests

All buttons currently link to the root path as a placeholder (real links added in next user stories)

## RSpec Results
```
39 examples, 0 failures

Coverage report generated for RSpec - 421 / 421 LOC (100.0%) covered.
```

### CODE REVIEW

File: spec/features/posts/index_spec.rb [hunk 1]
```diff
@@ -1,33 +1,34 @@
...
-    @day_3 = Day.create(week: 4, day_of_week: 3, group: @group_1)
-    @day_4 = Day.create(week: 6, day_of_week: 4, group: @group_1)
-    @post_1 = Post.create(body: "Wow, first day of the mod 1... syllabus day... but still so lost @.@", tone: "confuse", day: @day_1, user: @user)
-    @post_2 = Post.create(body: "Just took my first in class challenge. I'm NEVER going to get any better at this!!", tone: "angry", day: @day_2, user: @user)
-    @post_3 = Post.create(body: "Learning that I gotta utilize my resources and getting better doesn't come out of thin air. Make sure to use paired.tech!!", tone: "optimistic", day: @day_3, user: @user)
-    @post_4 = Post.create(body: "Passed Mod 1!! Time to get schwifty with the Turing crew", tone: "excited", day: @day_4, user: @user)
-    allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(@user)
-    visit '/'
-  end
+RSpec.describe 'Posts index -', type: :feature do
```
Comment (d-atkins) [inline spec/features/posts/index_spec.rb:3]: I like this format for describe blocks:
```
<name of page>
  <type of user>
    <assertions>
  <type of user>
    <assertions>
```

File: spec/features/posts/index_spec.rb [hunk 2]
```diff
@@ -1,33 +1,34 @@
...
+      @day_4 = Day.create(week: 6, day_of_week: 4, group: @group_1)
+      @post_1 = Post.create(body: 'Wow, first day of the mod 1... syllabus day... but still so lost @.@', tone: 'confuse', day: @day_1, user: @user)
+      @post_2 = Post.create(body: "Just took my first in class challenge. I'm NEVER going to get any better at this!!", tone: 'angry', day: @day_2, user: @user)
+      @post_3 = Post.create(body: "Learning that I gotta utilize my resources and getting better doesn't come out of thin air. Make sure to use paired.tech!!", tone: 'optimistic', day: @day_3, user: @user)
+      @post_4 = Post.create(body: 'Passed Mod 1!! Time to get schwifty with the Turing crew', tone: 'excited', day: @day_4, user: @user)
+      allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(@user)
+      visit posts_path
+    end
 
-  it "when I visit the root path, I click on a link to the posts index page" do
```
Comment (d-atkins) [inline spec/features/posts/index_spec.rb:20]: I'd put this kind of test in a dedicated nav test (and will make them later)

File: spec/features/posts/show_spec.rb [hunk 3]
```diff
@@ -1,40 +1,38 @@
...
-      visit '/posts'
+      Post.create(body: "Wow, first day of the mod 1... syllabus day... but still so lost @.@", tone: "confuse", day: day_1, user: user)
+      Post.create(body: "Just took my first in class challenge. I'm NEVER going to get any better at this!!", tone: "angry", day: day_2, user: user)
+      Post.create(body: "Learning that I gotta utilize my resources and getting better doesn't come out of thin air. Make sure to use paired.tech!!", tone: "optimistic", day: day_3, user: user)
+      @post_4 = Post.create(body: "Passed Mod 1!! Time to get schwifty with the Turing crew", tone: "excited", day: @day_4, user: user)
+      allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(user)
+      visit post_path(@post_4)
     end
 
-    it "can see a list of all posts belonging to that group" do
```
Comment (d-atkins) [inline spec/features/posts/show_spec.rb:21]: This was already tested in the index spec

File: spec/features/posts/show_spec.rb [hunk 4]
```diff
@@ -1,40 +1,38 @@
...
-      expect(page).to have_content("All Posts")
-      expect(page).to_not have_link("Click to see post #{@post_2.id}'s info.")
-      expect(page).to have_link("Click to see post #{@post_3.id}'s info.")
-      expect(page).to have_content(@post_3.body)
-    end
-
-    it "can click a link to see that post's information" do
```
Comment (d-atkins) [inline spec/features/posts/show_spec.rb:28]: I separated this into two separate test: 
1. clicking the link takes you where expected - I test this in the spec for the page you're starting from (in this case, the posts/index_spec)
2. checking the information displayed

File: app/views/posts/show.html.erb [hunk 5]
```diff
@@ -1,7 +1,13 @@
...
+<p><%= @post.user.group.name %></p>
+<p class='.day-week'><%= "#{@post.day.day_of_week}," %> Week <%= @post.day.week %></p>
 <p><%= @post.body %></p>
+
+<section id='reactions'>
+  <%= button_to 'Like', '/' %>
+  <%= button_to 'Dislike', '/' %>
+  <%= button_to 'Can relate', '/' %>
+  <%= button_to 'Lol', '/' %>
+  <%= button_to 'Upset', '/' %>
```
Comment (mronauli) [inline app/views/posts/show.html.erb:12]: clean

**Pull request merged**